draft-ietf-rtcweb-jsep.xml

```diff
@@ -1442,10 +1442,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               used for sending media, along with the mandatory "cname" source
               attribute, as specified in Section 6.1, indicating the CNAME for
               the source. The CNAME must be generated in accordance with
-              <xref target="RFC7022"></xref>. [OPEN ISSUE: How are CNAMEs
-              specified for MSTs? Are they randomly generated for each
-              MediaStream? If so, can two MediaStreams be synced?
-              See: https://github.com/rtcweb-wg/jsep/issues/4]</t>
+              <xref target="RFC7022"></xref>.
+              Each PeerConnection MUST use a different CNAME. All of the
```

Comment:
This guidance is buried pretty deeply here in this section that tries to explain how to generate an offer. Especially if we are going to need similar text in the answer, it might make sense to pull this guidance on CNAMEs out to its own 3.x subsection.

Then this text can just say to use the appropriate CNAME as indicated in that subsection.